hs/src/Reach/Simulator/Core.hs

```diff
@@ -28,6 +48,8 @@ initState = State
   , e_linstate = M.empty
   , e_nwtime = 0
   , e_nwsecs = 0
+  , e_locals = M.empty
```

Comment:
I find that using `mempty` is really convenient because if we change the representation it will just update without having to modify the code